webClient/src/SamplePage.tsx

```diff
@@ -49,17 +49,28 @@ class SamplePage extends React.Component<any, any> {
                   {/* Action types are used to determine what sort of Action is being taken on whatever App instance is the target. Launch simply creates a new instance with the context you provide, but Message can be used to communicate with an already open Instance to drive some action */}        
                   <label style={styles.boldText}>{t('action_type')}</label>
                   <input type="radio" value="Launch" name="actionType" checked={this.props.actionType == "Launch"} onChange={this.props.handleActionTypeChange}/>
-                  <label htmlFor="actionLaunch">{t('launch')}</label>
+                  <label  style={styles.labelPadding} htmlFor="actionLaunch">Launch</label>
                   <input type="radio" value="Message" name="actionType" checked={this.props.actionType == "Message"} onChange={this.props.handleActionTypeChange}/>
-                  <label htmlFor="actionMessage">{t('message')}</label>
+                  <label  style={styles.labelPadding} htmlFor="actionMessage">Message</label>
+                  <input type="radio" value="Maximize" name="actionType" checked={this.props.actionType == "Maximize"} onChange={this.props.handleActionTypeChange}/>
+                  <label  style={styles.labelPadding} htmlFor="actionLaunch">Maximize</label>
```

Comment:
These likewise should be `t('maximize')` and so on. We will need to make translations soon after.